pkgs/by-name/up/upsun/package.nix

```
@@ -0,0 +1,36 @@
+{ stdenvNoCC, lib, fetchurl }:
+
+let
+  os = "linux";
+  arch = if stdenvNoCC.isAarch64 then "arm64" else "amd64";
+  platform = "${os}_${arch}";
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "upsun";
+  version = "5.0.12";
+
+  src = fetchurl {
+    url = "https://github.com/platformsh/cli/releases/download/${version}/upsun_${version}_${platform}.tar.gz";
+    sha256 = {
+      x86_64-linux = "sha256-svEPMVY7r7pAoXwFIMYqCEduqR3Nkocaguf2nIGt+G8=";
+      aarch64-linux = "sha256-ZraS/PqSPL/kcj5o6hzDdL70IV2IWXOma6OHCiXIDQc=";
+    }.${stdenvNoCC.system} or (throw "${pname}-${version}: ${stdenvNoCC.system} is unsupported.");
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  sourceRoot = ".";
+  installPhase = ''
+    install -Dm755 upsun $out/bin/upsun
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Either use `lib.foo`, or `with lib; foo`, don't mix them. Of course, it's better to write `lib.foo`